src/components/create-from-builder/create-from-builder.controller.ts

```diff
@@ -144,9 +144,6 @@ export class CreateFromBuilderController implements angular.IController {
     this.noProjectsCantCreateWatch = this.$scope.$on('no-projects-cannot-create', () => {
       this.ctrl.noProjectsCantCreate = true;
     });
-
-    this.getServiceClassesAndPlans();
-    this.instancesSupported = !!this.APIService.apiInfo(this.APIService.getPreferredVersion('serviceinstances'));
```

Comment:
There is a `if (this.ctrl.showPodPresets)` block above (line 136) that does this when pod presets are enabled. I think this was an oversight that it was left here, but it's not needed.